client/components/NetworkForm.vue

```diff
@@ -167,6 +167,16 @@
 					maxlength="300"
 				/>
 			</div>
+			<div class="connect-row">
+				<label for="connect:leaveMessage">Away message</label>
+				<input
+					id="connect:leaveMessage"
+					v-model="defaults.leaveMessage"
+					class="input"
+					name="leaveMessage"
+					placeholder="Smell ya later"
```

Comment:
I feel like something a bit more serious (like a simple "Bye!") would be more appropriate here.